### ISSUE TITLE
[Assistive tech and device support] Focus state isn't visible. (11.09.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29335]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Trevor Pierce (Accessibility)

---

### Platform Issue
Focus state isn't visible.

### Issue Details
Many of the links, buttons, and form inputs are missing our yellow focus halo.

### VA.gov Experience Standard
[Cateogy Number 11, Issue Number 09](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.7_AA

---

### Platform Recommendation
All user-focusable elements (links, buttons, form inputs, etc.) must show the yellow focus halo when the element has focus.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by sonntag-adam** [comment]
@shiragoodman this is in staging. ready for review.

**Comment by sonntag-adam** [comment]
@noahgelman this is in staging. ready for review.

**Comment by shiragoodman** [comment]
hi @sonntag-adam  - it is the VFS teams responsibility to validate fixes identified by the Platform. We will not be reviewing this to verify the fix was made correctly. If you have questions, you are welcome to bring them to Collab Cycle Office Hours on Wednesday 11:30am-12pm ET.

**Comment by WillMcCormack** [comment]
@shiragoodman how do we close out these tickets? We met with @noahgelman and Trevor on Friday and they asked us to tag them when the fixes were ready so that they could review and close the tickets.

**Comment by shiragoodman** [comment]
hey @WillMcCormack. We agreed to doing a second Staging Review when your team is ready. At that time, we will validate the fixes. Please keep these tickets open until them. If they're resolved, we can close them out.

**Comment by WillMcCormack** [comment]
@shiragoodman thanks for clarifying.

**Comment by mattpointzxer0** [comment]
@1Copenut and @noahgelman; this is awaiting your review. Appreciate it!

**Comment by dneel-bah** [comment]
Per agreement with OIT/OCTO PMs this issue passed internal testing with assistive technology.  Closing this issue as completed
